### DISCUSSION TITLE
邮箱无法收到恢复码

### DISCUSSION THREAD

**Discussion by LuluJinx** [getlantern/lantern-forum-cn#515] [Q&A] [answered]
4月升级到专业版，昨天偶然发现账户邮箱并非自己的邮箱；
重新通过邮箱授权，但收不到恢复码；
请发送恢复码到347872853@qq.com
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@LuluJinx 您好. 请您也提供我支付单据. 还有提供我您现在看到不是您的那个邮箱!

**Comment by LuluJinx** [comment]
***@***.***；


另外，支付单据已添加在附件中。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月18日(星期六) 凌晨1:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 邮箱无法收到恢复码 (Discussion #515)





 
@LuluJinx 您好. 请您也提供我支付单据. 还有提供我您现在看到不是您的那个邮箱!

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/bbbeefd0-e9bf-4626-8d2b-f30eeaecc03c)

@LuluJinx 您好. 我这里收不到, 也看不到. 好像被github屏蔽了. 您可以发邮件给我吗? 然后告诉我您是用那个邮件发.

**Reply by LuluJinx** [reply]
@Mike777Khoo 你好，因为已经解除了设备的绑定关系，仅凭印象和搜索记录，那个非我的邮箱似乎是[47033359@qq.com](mailto:47033359@qq.com)；

另外，支付单据已添加在附件中。
![支付信息](https://github.com/getlantern/lantern-forum-cn/assets/166150244/725ccda6-c329-4bad-b561-ea5757f1d5c0)

**Comment by MaxMKMax** [comment]
@LuluJinx 关于您遇到的问题我们深感抱歉! 
好的. 谢谢您. 我这里帮您手动提交问题.

**Comment by MaxMKMax** [comment]
@LuluJinx 已经帮您提交上去了. 请给我们几天的工作日. 会尽快给您恢复帐号. 过几天请查看邮箱或Github. 谢谢您的等待!
谢谢您支持蓝灯.

**Comment by MaxMKMax** [comment, marked as answer]
@LuluJinx 
您好. 这是您专属的激活码:
F7PFH-6JFVW-RPXD2-4CJ9C-KJYVX

**Comment by tyf9911** [comment]
您好，我有着同样的问题在一周前曾经在别的贴子下面反馈过然后就没有了消息。麻烦您能够处理下发下邀请码，到389919165@qq.com
![2c7a373bae1f630683fd4a8d9622d5b](https://github.com/getlantern/lantern-forum-cn/assets/55548465/d837fc8a-0b56-4250-9037-c5e5486ff935)
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@tyf9911 您好. 已经帮您提交问题了. 请稍等.

**Comment by ayydu** [comment]
@LuluJinx 我的也是同样的毛病麻烦您能够处理下发下邀请码157751104@qq.com
![QQ图片20240522141837](https://github.com/getlantern/lantern-forum-cn/assets/170499416/5a24cc50-1d16-4346-8ccf-28f28b2e1a30)

**Reply by MaxMKMax** [reply]
@ayydu 您好. 好的. 我帮您问一下激活码. 不好意思我最近晚上才会上线, 因为要跟上美国时间.

**Comment by samsgood0310** [comment]


**Reply by MaxMKMax** [reply]
@samsgood0310 您好! 好的, 我会帮您提交问题然后让技术人员帮您修复. 技术人员需要您看到的那个不是你的邮箱. 请提供那个邮箱给我.

**Reply by samsgood0310** [reply]
@Mike777Khoo 你好，我怀疑你提供的支付宝支付链接是有问题的，充值的都是一个账号，然后会覆盖前面的账号，我的账号一开始充值的时候是对的，可以通过邮箱获取验证码，后来账号不对了，就获取不了了，现在显示的账号是feiruimail@sina.com

**Reply by MaxMKMax** [reply]
@samsgood0310 
您好. 这是您的激活码. 可以用它重新登入您的专业版:
VW676-TFJMD-MTJYR-JP4G9-RRV9D

**Reply by samsgood0310** [reply]
@Mike777Khoo ,467211319@qq.com这个账号又被顶替掉了，也收不到邮箱验证码，不能使用的话，你就给我退款吧，才用几天就这样

**Comment by hrk0321** [comment]
@Mike777Khoo 您好，我也有相同的问题。从2月开始，页面就一直显示为其他人的专业版账号邮箱，而且经常变成不同的账号。尝试过在“支持-报告问题”那里反馈，但没有得到解决。今天发现不会显示其他人的账号了，但是自己的邮箱依然无法收到验证码，请处理一下。我的注册邮箱是715745830@qq.com
<img width="540" alt="lantern" src="https://github.com/getlantern/lantern-forum-cn/assets/170598160/a1591204-b9cf-4769-b495-06b5ebadc702">

**Reply by MaxMKMax** [reply]
@hrk0321 您好! 好的, 我会帮您提交问题然后让技术人员帮您修复. 技术人员需要您看到的那个不是你的邮箱. 请提供那个邮箱给我.

**Reply by hrk0321** [reply]
第一次看到的是815357956@qq.com，之后看到的其他邮箱没有记录下来。

**Reply by MaxMKMax** [reply]
@hrk0321 您好. 这是您的激活码. 可以用它重新登入您的专业版:
8GFBP-T9FGF-9CDQ4-WKM2H-KH9GM

**Comment by bucengfangqi** [comment]
@Mike777Khoo 
我今天偶然发现账户邮箱并非自己的邮箱；
重新通过邮箱授权，但收不到恢复码；
请发送恢复码到[15088672074@163.com](mailto:15088672074@163.com)
![49149f9534b921fcb925b9075af72e6](https://github.com/getlantern/lantern-forum-cn/assets/22071679/a11b5ec0-ef98-4a6d-b830-b74e0a052568)

**Reply by MaxMKMax** [reply]
@bucengfangqi 您好! 好的, 我会帮您提交问题然后让技术人员帮您修复. 技术人员需要您看到的那个不是你的邮箱. 请提供那个邮箱给我.

**Reply by bucengfangqi** [reply]
feiruimain@sina.com

**Reply by bucengfangqi** [reply]
@Mike777Khoo  所以？这个问题什么时候能解决？我这边等着用呢

**Reply by MaxMKMax** [reply]
@bucengfangqi 
您好. 这是您的激活码. 可以用它重新登入您的专业版:
CJJ6X-2RH2D-XMPDX-DJKFX-J6TF7

**Comment by bucengfangqi** [comment]
> @bucengfangqi 您好! 好的, 我会帮您提交问题然后让技术人员帮您修复. 技术人员需要您看到的那个不是你的邮箱. 请提供那个邮箱给我.

[feiruimain@sina.com](mailto:feiruimain@sina.com)

**Comment by tyf9911** [comment]
您好我收到贵司发我的激活码，但请问贵司是否是之前的记录已经全部丢失，我当时是还有剩余的一定时间的，且一次性的续费了3次，总计时间大约10年。但激活码恢复只有两年的时间。附件是我最近三次的付款记录及目前的使用期限。




***@***.***
 
发件人： Mike K
发送时间： 2024-05-23 00:56
收件人： getlantern/lantern-forum-cn
抄送： tyf9911; Mention
主题： Re: [getlantern/lantern-forum-cn] 邮箱无法收到恢复码 (Discussion #515)
@tyf9911 您好. 已经帮您提交问题了. 请稍等.

**Comment by tyf9911** [comment]
***@***.******@***.***邮箱是同一个使用者




***@***.***
 
发件人： ***@***.***
发送时间： 2024-05-24 10:24
收件人： getlantern/lantern-forum-cn
主题： Re: Re: [getlantern/lantern-forum-cn] 邮箱无法收到恢复码 (Discussion #515)
您好我收到贵司发我的激活码，但请问贵司是否是之前的记录已经全部丢失，我当时是还有剩余的一定时间的，且一次性的续费了3次，总计时间大约10年。但激活码恢复只有两年的时间。附件是我最近三次的付款记录及目前的使用期限。




***@***.***
 
发件人： Mike K
发送时间： 2024-05-23 00:56
收件人： getlantern/lantern-forum-cn
抄送： tyf9911; Mention
主题： Re: [getlantern/lantern-forum-cn] 邮箱无法收到恢复码 (Discussion #515)
@tyf9911 您好. 已经帮您提交问题了. 请稍等.

**Reply by tyf9911** [reply]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/5b426ce6-ce76-43d7-9bce-6749f80e71ab)
@tyf9911 您好. GitHub屏蔽您的信息了. 我这里看不到. 请您把信息发到我的邮件, 包括支付单据和账号邮箱. 

我同时也会帮您确认好您的专业版账号的时间.

您放心, 我会帮您处理好. 谢谢您支持蓝灯! 

发邮件到: mike@getlantern.org

**Comment by MaxMKMax** [comment]
@tyf9911 您好. 已经帮您加了 10 年的专业班时间了. 请您帮我一个忙, 提供我您的所有支付单据. 发到GitHub或Email我:
mike@getlantern.org
发完了之后告诉我您用哪个邮箱发的. 谢谢哦!

**Comment by bhu1987** [comment]


**Reply by MaxMKMax** [reply]
@samsgood0310 您好. 您理解错误了. 账号变成别的邮箱是别的问题. 不是支付链接的问题. 技术团队正在修复了. 
您给我们一点时间. 我来帮您解决问题

**Comment by weizijian** [comment]
@Mike777Khoo   你好 6月13号升级到专业版 邮箱授权收到不邮件 893586239@qq.com

**Reply by MaxMKMax** [reply]
@weizijian 您好. 请您也提供我您的支付单据. 我帮您提交问题.
